capstone-frontend/src/styles/Login.css

```diff
@@ -1,19 +1,13 @@
-/* Logout Dropdown button */
-#login div div button {
-  background: none;
-  border: none;
-  margin: 0px;
-  padding: 0px;
-  text-align: center;
+#login-card {
+  width: 45em;
+  margin: 0 auto;
+  transform: translateY(50%);
 }
 
-/* Login Button */
-#login div button {
-  background: none;
-  border-width: 0px !important;
-  margin: 0px;
+#sign-in-text span {
+  font-weight: 400;
 }
 
-#profile_img {
-  width: 35%;
+#sign-in-text button{
+  font-size: 18px !important;
```

Comment:
Is there any way to override this without using the !important here? For example using the most specific selector possible?

Reply:
I don't think that possible. In this scenario, I am trying to change the styling of a pre-made component. Without the ```!important``` my changes have no effect.